README.md

```diff
@@ -1,2 +1,15 @@
-# ux-widget-framework
-Library and tooling for Modular WIdgets
+# Blue Ocean Modular UX Widgets
+
+## Packages
+
+-   **[ux-widget-framework](packages/ux-widget-framework)** - Contains both build-time tools to compile widgets, as well as runtime library code used by widgets themselves.
+-   **[pipeline-graph](packages/pipeline-graph)** - The PipelineGraph component used to show the results of a build in graphical form, from Blue Ocean.
+
+## Building
+
+If you don't have a global install of the `lerna` CLI tool, use `npx lerna` instead.
```

Comment:
Just a note, `npx lerna` will fail - if the user doesn't have it installed globally. If you move `dist` or any other task to the parent `package.json` as a script ` { "dist": "lerna run dist" } ` then doing `npm run dist` - will automatically try to use local otherwise fallback to the global binary.

Reply:
Isn't the whole point of `npx` that it will go and fetch the tool if you don't have it? Maybe I'm confused, I'll run some more tests.